README.md

```diff
@@ -1,24 +1,46 @@
-### Salesforce ESB Connector
+# Salesforce EI Connector
 
-The Salesforce connector allows you to access the Salesforce API through WSO2 ESB. Salesforce is  a web-based service
+The Salesforce [Connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the [Salesforce SOAP API](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm) through WSO2 EI. Salesforce is  a web-based service
```

Comment:
The Salesforce [Connector](https://docs.wso2.com/display/EI640/Working+with+Connectors) allows you to access the [Salesforce SOAP API](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm) through WSO2 Enterprise Integrator (WSO2 EI).